.changeset/angry-mugs-invent.md

```diff
@@ -0,0 +1,26 @@
+---
+'@graphql-inspector/introspect-command': major
+'@graphql-inspector/commands': major
+'@graphql-inspector/coverage-command': major
+'@graphql-inspector/validate-command': major
+'@graphql-inspector/similar-command': major
+'@graphql-inspector/graphql-loader': major
+'@graphql-inspector/loaders': major
+'@graphql-inspector/audit-command': major
+'@graphql-inspector/serve-command': major
+'@graphql-inspector/github-loader': major
+'@graphql-inspector/diff-command': major
+'@graphql-inspector/docs-command': major
+'@graphql-inspector/code-loader': major
+'@graphql-inspector/json-loader': major
+'@graphql-inspector/git-loader': major
+'@graphql-inspector/url-loader': major
+'@graphql-inspector/action': major
+'@graphql-inspector/config': major
+'@graphql-inspector/logger': major
+'@graphql-inspector/core': major
+'@graphql-inspector/cli': major
+'@graphql-inspector/ci': major
+---
+
+Upgrade pnpm, bump Node engine and Yoga to version 5
```

Comment:
- `pnpm` is not relevnat here, it's a dev tool for us. can be removed
- please specify explicitly that node 18 is the minimal version required
- Yoga ✅